components/member_list_channel/member_list_channel.jsx

```diff
@@ -9,7 +9,7 @@ import * as UserAgent from 'utils/user_agent.jsx';
 
 import ChannelMembersDropdown from 'components/channel_members_dropdown';
 import SearchableUserList from 'components/searchable_user_list/searchable_user_list_container.jsx';
-
+import LoadingScreen from 'components/loading_screen.jsx';
```

Comment:
```suggestion
import LoadingScreen from 'components/loading_screen';
```

This component was recently migrated to typescript.